### DISCUSSION TITLE
Questions about the number of TLS 1.3 session tickets

### DISCUSSION THREAD

**Discussion by baoyi84930** [openssl/openssl#25090] [Q&A]
I noticed that there are multiple scenarios for the number of NST sent by the server.
 According to the manual:
> Following a resumption the number of tickets issued will never be more than 1 regardless of the value set via SSL_set_num_tickets() or SSL_CTX_set_num_tickets(). 

**The first question**: I don't quite understand why. Is it for any security reasons?

In addition, I also noticed that when the server attempts session recovery but does not match the hash algorithm, a certificate handshake will be performed, and then the number of tickets is 0.
**The second question**: Why not 2(Certificate authentication), but 0？

**Comment by tom-cosgrove-arm** [comment]
Given this is a question, shouldn't it rather be a discussion?
